pkg/agent/core/ngt/handler/grpc/handler.go

```diff
@@ -23,20 +23,28 @@ import (
 	"strconv"
 
 	agent "github.com/vdaas/vald/apis/grpc/agent/core"
+	"github.com/vdaas/vald/apis/grpc/gateway/vald"
 	"github.com/vdaas/vald/apis/grpc/payload"
+	"github.com/vdaas/vald/internal/errgroup"
 	"github.com/vdaas/vald/internal/errors"
 	"github.com/vdaas/vald/internal/info"
 	"github.com/vdaas/vald/internal/log"
 	"github.com/vdaas/vald/internal/net/grpc"
 	"github.com/vdaas/vald/internal/net/grpc/status"
 	"github.com/vdaas/vald/internal/observability/trace"
+	"github.com/vdaas/vald/internal/safety"
 	"github.com/vdaas/vald/pkg/agent/core/ngt/model"
 	"github.com/vdaas/vald/pkg/agent/core/ngt/service"
 )
 
-type Server agent.AgentServer
+type Server interface {
+	agent.AgentServer
+	vald.ValdServer
```

Comment:
it seems not to implement v1 APIs. is it okay? :thinking:

Reply:
I'm looking into it, please wait a moment